cmd/files/pull/cmd_pull.go

```diff
@@ -68,11 +69,14 @@ Available options:
   --source
     Download source files along with translated files.
 
+  --all
+    Download all files.
```

Comment:
Does it work for translated and source files? I added the `translated` word, remove it if not required.
```suggestion
    Download all translated files. Required if no file pattern is specified.
```